spotless task doesn't exist

**Describe the bug**
When running the `./gradlew spotless` that is mentioned in the [CONTRIBUTING.md](https://github.com/open-telemetry/opentelemetry-java/blob/master/CONTRIBUTING.md), I get an error.

**Steps to reproduce**
Run `./gradlew spotless`.

**What did you expect to see?**
I expected to be told if my code met the verification steps.

**What did you see instead?**
```log
FAILURE: Build failed with an exception.

* What went wrong:
Task 'spotless' is ambiguous in root project 'opentelemetry-java'. Candidates are: 'spotlessApply', 'spotlessCheck', 'spotlessDiagnose', 'spotlessInternalRegisterDependencies', 'spotlessJava', 'spotlessJavaApply', 'spotlessJavaCheck', 'spotlessJavaDiagnose'.
```

**What version and what artifacts are you using?**
The latest code on master for `opentelemetry-java`.

**Environment**
Compiler: openjdk 11.0.8 2020-07-14 LTS
OS: macOS Catalina (10.15.6)

**Additional context**
I believe it's just a quick update to the CONTRIBUTING.md page, changing the `spotless` to `spotlessCheck`. But is worth someone verifying.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -64,7 +64,7 @@ $ ./gradlew spotlessApply
 To verify code style manually run the following command, 
 which uses [google-java-format](https://github.com/google/google-java-format) library:
 
-`./gradlew spotless`
+`./gradlew spotlessCheck`
 
 ### Best practices that we follow
 

